source/schemas/types.txt

```diff
@@ -509,3 +509,124 @@ with the ``title`` field set to the embedded object's type name:
             }
       }
    }
+
+.. _geospatial:
+
+Geospatial Data
+---------------
+
+Geospatial data describes points and other data on the earth's surface. App 
+Services does not have built-in geospatial types. Instead, you model geographic
+data using standard GeoJSON objects.
+
+If you are not using Device Sync, you can query your geospatial data with regular 
+:manual:`geospatial operators </geospatial-queries>`. However, if you are 
+using Device Sync, there are additional requirements in how you store and 
+query the data.
+
+GeoJSON Point
+~~~~~~~~~~~~~
+
+A GeoJSON Point (GeoPoint) is a single location on the Earth's surface. The GeoPoint 
+schema must a **required** ``type`` field, of type string, which is always set 
+to "Point".
+
+You must also provide a ``coordinates`` field, which is an array of doubles. 
+The ``coordinates`` array must contain  at least 2 double values, and may 
+contain a third:
+    
+- The first double is the longitude of the point, between -180 and 180.
+- The second is the latitude, between -90 and 90.
+- The optional third value represents the elevation/altitude of the point, 
+  in meters. Atlas ignores this value when performing queries. 
+
+.. note::
+
+   You cannot set an array as required in a Device Sync schema, so the Sync 
+   server checks that the coordinates field is present and meets the 
+   requirements.
+
+To store a GeoPoint in Atlas for later querying with :ref:`Atlas Device Sync <sync>`, 
+the following rules also apply to the GeoPoint object:
+
+- It must be of type ``object``
+- It must be embedded within another type
+- Additional properties are allowed within the ``coordinates`` array, as long as 
+  they are doubles.
+- Additional properties in the GeoPoint object are also allowed.
+ 
+The following is the Device Sync schema of an object with an embedded GeoPoint 
+property named "location":
+
+.. code-block:: json
+   :copyable: false
+
+   {
+     "title": "MyObject",
+     "properties": {
+        "location": {
+          "bsonType": "object",
+          "required": [ "type" ],
+          "properties": {
+            "type": {
+              "bsonType": "string",
+            },
+            "coordinates": {
+              "bsonType": "array",
+              "items": {
+                "bsonType": "double"
+              }
+            }
+          }
+        }
+      }
+   }
+
+The following code block shows a document that follows the schema. It has a 
+single embedded GeoPoint object named "location". Both the containing object 
```

Comment:
[nit] I don't think this statement is accurate anymore (just the containing object has an extra property)